samples/SunburstChartExample/README.md

```diff
@@ -0,0 +1,23 @@
+# Issue 674 PPTX Sunburst chart? 
+
+The sample code in this folder (program.cs) is an example of how to generate the Office 2016 Sunburst chart type using the SDK. The bulk of the code is generated from the OpenXML SDK Productivity Tool, reflecting from a pptx deck that had a simple bar chart. Then it was modified to generated the extended chart part for the Sunburst chart type.
+
+This relies on the code fix added in PR #687. Without that, the chartSpace and chart elements won't connect up to the chart part correctly.
```

Comment:
Again, don't refer to a PR. You could refer to a commit, but only one that has already been added to master. Since this will be available in 2.11.0, you could reference that as the baseline requirement.

Reply:
removed